src/config.rs

```diff
@@ -34,6 +34,8 @@ pub enum RawColor {
     Purple,
     Cyan,
     White,
+    ANSI(u8),
```

Comment:
Rust style guides would generally suggest `Ansi` here.

```suggestion
    Ansi(u8),
```